package.json

```diff
@@ -12,6 +12,7 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "axios": "^1.2.4",
+    "env-cmd": "^10.1.0",
```

Comment:
Let's remove this node module. I understand that its better to have env files for different environment but for our scale of the app, there's really no need to have two different env variable files. 

Let's only have one file .env for now.